ASTree.cpp

```diff
@@ -2474,7 +2474,8 @@ PycRef<ASTNode> BuildFromCode(PycRef<PycCode> code, PycModule* mod)
             }
             break;
         default:
-            fprintf(stderr, "Unsupported opcode: %s\n", Pyc::OpcodeName(opcode & 0xFF));
+            int bytecode = code->code()->value()[curpos] & 0xFF;
```

Comment:
Reading directly from the code object doesn't handle the `EXTENDED_ARG` case properly...  Perhaps just printing the position is sufficient, since it can be used to look up the context in disassembly.